app/graphql/mutations/demarche_ajouter_administrateur.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Mutations
+  class DemarcheAjouterAdministrateur < Mutations::BaseMutation
+    description "Ajouter un administrateur a une démarche"
+
+    argument :demarche, Types::DemarcheDescriptorType::FindDemarcheInput, "La démarche", required: true
+    argument :administrateurs, [Types::ProfileInput], "Administrateur à ajouter.", required: true
+
+    field :demarche, Types::DemarcheDescriptorType, null: true
+    field :errors, [Types::ValidationErrorType], null: true
+    field :warnings, [Types::WarningMessageType], null: true
+
+    def resolve(demarche:, administrateurs:)
+      demarche_number = demarche.number.presence || ApplicationRecord.id_from_typed_id(demarche.id)
+      demarche = Procedure.find_by(id: demarche_number)
+
+      ids, emails = partition_administrators_by(administrateurs)
```

Comment:
si t'aimes bien la suggestion d'avant : 
```suggestion
      ids, emails = partition_administrators_by_profile_input(administrateurs)
```